pkgs/development/python-modules/mediafile/default.nix

```diff
@@ -0,0 +1,29 @@
+{ buildPythonPackage
+, enum34
+, fetchPypi
+, lib
+, pythonOlder
+, pythonPackages
```

Comment:
don't import pythonPackages while in `python-modules, just import the individual packages
```suggestion
```